utilities.html

```diff
@@ -0,0 +1,62 @@
+<!doctype html>
+<html>
+<head>
+<meta charset="utf-8" />
+<title>The Mx Thing - Utilities</title>
+<link rel="stylesheet" type="text/css" href="style.css"/>
+<meta name="viewport" content="width=device-width, initial-scale=1"/>
+</head>
+<body>
+  <a href="index.html">Home Page</a>
+<h1>Utilities</h1>
+
```

Comment:
@kittsville Thanks for this request. It looks like the company name is missing here (for lines 13-17)? Otherwise, this pull request looks good.

Reply:
My bad! I was using it as a template then forgot to delete it. Proof I shouldn't code in the evening 😅